altair/vegalite/v2/api.py

```diff
@@ -416,8 +416,10 @@ def transform_filter(self, filter, **kwargs):
         return self._add_transform(core.FilterTransform(**kwargs))
 
     @utils.use_signature(core.LookupTransform)
-    def transform_lookup(self, *args, **kwargs):
-        return self._add_transform(core.LookupTransform(*args, **kwargs))
+    def transform_lookup(self, from_=Undefined, **kwargs):
+        if from_ is not Undefined:
+            kwargs['from'] = from_
```

Comment:
Just to cover our bases, let's raise a ValueError if ``'from'`` is already in kwargs

```
if from_ is not Undefined:
    if 'from' in kwargs:
        raise ValueError("transform_lookup: both 'from_' and 'from' passed as arguments.")
    # ...
```